mapie/tests/test_quantile_regression.py

```diff
@@ -159,24 +159,17 @@ def test_no_predict_fit_estimator() -> None:
 
 def test_no_para_loss_estimator() -> None:
     """Test to check when it does not have a valid loss_name."""
-    Params = TypedDict(
```

Comment:
You can leave `TypedDict` dictionaries in the tests since they are used with `mapie-dev` conda environment that includes `typing-extensions`.

Reply:
Yes, I would normally agree but in this case, for typing measures, we are also changing these since their role is to imitate an estimator. Therefore the dictionary entries needs to match with that of the class `MapieQuantileRegressor`.

Reply:
Yes, you're right !